fh-dataman.json

```diff
@@ -0,0 +1,238 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1.0.0",
+  "metadata": {
+    "name": "fh-dataman",
+    "annotations": {
+      "description": "Red Hat Mobile Databrowser Backend Service template",
+      "iconClass": "icon-nodejs"
+    }
+  },
+  "objects": [
+    {
+      "kind": "Service",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "fh-dataman-service",
+        "labels": {
+          "name": "fh-dataman-service"
+        },
+        "annotations": {
+          "service.alpha.openshift.io/dependencies": "[{\"name\":\"mongodb-1\",\"namespace\":\"\",\"kind\":\"Service\"},{\"name\":\"fh-messaging-service\",\"namespace\":\"\",\"kind\":\"Service\"},{\"name\":\"fh-metrics-service\",\"namespace\":\"\",\"kind\":\"Service\"},{\"name\":\"fh-statsd-service\",\"namespace\":\"\",\"kind\":\"Service\"}]"
```

Comment:
OpenShift annotation on the project to group related services. See https://docs.openshift.com/container-platform/3.5/dev_guide/templates.html#other-recommendations